docs/semgrep-assistant/overview.md

```diff
@@ -49,42 +49,42 @@ By categorizing your code through component tags, Semgrep Assistant can help you
 
-Semgrep presents these recommendations through Slack and PR or MR comments, so developers can fix or triage an issue without switching contexts.
+Auto-triage recommendations are available in Semgrep Cloud Platform's main **Findings page** when filtering to findings that Assistant suggests should be ignored, and in the [finding's details](/docs/semgrep-code/findings/#view-findings-details-about-a-specific-finding). 
 
-![Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
-    *Figure*. Semgrep Assistant in a Slack notification.
+![Semgrep Assistant in the filtered Findings page](/img/semgrep-assistant-autotriage-findings.png)
+
+Assistant's suggestions to ignore findings are also sent through Slack and PR or MR comments, so developers can triage an issue without switching contexts.
 
-You can view findings that Assistant suggests should be **Ignored** in Semgrep Cloud Platform's **Findings page**. <!-- #10438 -->
+![Semgrep Assistant in a Slack notification](/img/semgrep-assistant-slack.png#md-width)
+    *Figure*. Semgrep Assistant auto-triage in a Slack notification.
 
 ### Autofix Semgrep Code findings
 
-Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings identified using rules that don't have human-written autofix suggestions when it identifies a true positive. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, then Assistant will recommend its use in the autofix suggestion for the issue in your code.
+Semgrep Assistant can suggest [autofix](/writing-rules/autofix/) code snippets for Semgrep Code findings when it identifies a true positive, if the rule doesn't have a human-written autofix. Assistant customizes the code snippets it provides based on previous feedback, if any, and your rule customizations. For example, if you have a custom rule that recommends a specific sanitizer, Assistant can recommend its use in the autofix suggestion for the issue in your code.
 
-You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant.
+You can set the minimum autofix confidence level required to display autofix suggestions from Semgrep Assistant in the **Settings** page in Semgrep Cloud Platform.
 
-This feature is only active in PR and MR comments. Developers can review and verify Semgrep's generated fixes before they're applied.
+Autofixes are available in Semgrep Cloud Platform's **Findings page** under **Your code** in the [finding's details](/docs/semgrep-code/findings/#view-findings-details-about-a-specific-finding). They are also available in PR and MR comments, so developers can review and verify Semgrep's generated fixes before they're applied.
```

Comment:
```suggestion
Autofixes are <!-- available in Semgrep Cloud Platform's **Findings** page under **Your code** in the [finding's details](/docs/semgrep-code/findings/#view-findings-details-about-a-specific-finding). They are also --> available in PR and MR comments, so developers can review and verify Semgrep's generated fixes before they're applied.
```

I saw that the [fix for this is high priority](https://linear.app/semgrep/issue/AI-505/autofix-not-showing-in-finding-details-for-findings-even-if-they-have), so I suggested we comment it out for the time being

Reply:
I'm honestly kinda hoping it'll be fixed before this is merged, but I'll leave this suggestion in case it makes more sense to merge earlier.

Reply:
This is fixed now, yay!